src/scss/_variables.scss

```diff
@@ -11,3 +11,6 @@ $sp-l: 18px;
 
 // sidebar
 $drawer-width: 240px;
+
+//sizes
+$container-size: 700px;

```

Comment:
Perhaps, it is better to use `width` instead of `size` and combine it with the previous `$drawer-width`?